frontend/src/scenes/data-warehouse/editor/QueryPane.tsx

```diff
@@ -87,6 +87,8 @@ export function QueryPane(props: QueryPaneProps): JSX.Element {
                             onMaxOpen={() => {
                                 reportAIQueryPromptOpen()
                             }}
+                            headline="What data do you want to analyze?"
+                            description="On this page, I can help you write and tweak SQL queries."
```

Comment:
Wouldn't work properly if two different max tools would be added to the same scene as one would override the other, but not a problem right now, just thought to mention it.